README.md

```diff
@@ -49,6 +52,31 @@ be sourced in your `.bashrc` or `.cshrc` which will make `xtb` and the
 parameter files available. Also a pkg-config file is generated to make
 `xtb` available in other projects as dependency.
 
+### CMake
+
+The CMake build system requires both `make` and `cmake` to be installed, the
+latter has to be version 3.9 or newer.
+
+Building XTB with CMake works with the following chain of commands:
+
+```bash
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/some/place/nice ..
```

Comment:
This command *will* be copied and pasted straight to the command line. I would prefer
```
cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PWD/install
```
or
```
cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$HOME/.local
```